compiler/ballerina-lang/spotbugs-exclude.xml

```diff
@@ -325,4 +325,7 @@
         <Class name="io.ballerina.projects.CompilerPluginManager"/>
         <Bug pattern="BC_UNCONFIRMED_CAST"/>
     </Match>
+    <Match>
+        <Package name="org.wso2.ballerinalang.compiler.nballerina"/>
```

Comment:
what are the spotbugs you are getting ?

Reply:
It doesn't allow to create a classloader, I'll try using a doPrivileged block instead

Reply:
if creating the classloader is the only issue, then it's ok. can you paste exact error please.

Reply:
I manged to fix it using a doPrivileged block, we no longer need to exclude the file